src/storage/index/LookUpIndexProcessor.cpp

```diff
@@ -1,20 +1,21 @@
-/* Copyright (c) 2019 vesoft inc. All rights reserved.
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
  *
  * This source code is licensed under Apache 2.0 License,
  * attached with Common Clause Condition 1.0, found in the LICENSES directory.
  */
 
-#include "LookUpEdgeIndexProcessor.h"
+#include "LookUpIndexProcessor.h"
 
 namespace nebula {
 namespace storage {
 
-void LookUpEdgeIndexProcessor::process(const cpp2::LookUpIndexRequest& req) {
+void LookUpIndexProcessor::process(const cpp2::LookUpIndexRequest& req) {
     /**
      * step 1 : prepare index meta and structure of return columns.
      */
     auto ret = prepareRequest(req);
     if (ret != cpp2::ErrorCode::SUCCEEDED) {
+        LOG(ERROR) << "Prepare Request " << "";
```

Comment:
`LOG(ERROR) << "Prepare Request failed"`

Reply:
also print the ret code ?